backend/src/graphql/resolver/CommunityResolver.ts

```diff
@@ -1,58 +1,18 @@
 import { Resolver, Query, Authorized } from 'type-graphql'
 
 import { Community } from '@model/Community'
+import { Community as DbCommunity } from '@entity/Community'
 
 import { RIGHTS } from '@/auth/RIGHTS'
-import CONFIG from '@/config'
 
 @Resolver()
 export class CommunityResolver {
-  @Authorized([RIGHTS.GET_COMMUNITY_INFO])
-  @Query(() => Community)
-  getCommunityInfo(): Community {
-    return new Community({
-      name: CONFIG.COMMUNITY_NAME,
-      description: CONFIG.COMMUNITY_DESCRIPTION,
-      url: CONFIG.COMMUNITY_URL,
-      registerUrl: CONFIG.COMMUNITY_REGISTER_URL,
-    })
-  }
-
   @Authorized([RIGHTS.COMMUNITIES])
   @Query(() => [Community])
-  communities(): Community[] {
-    if (CONFIG.PRODUCTION)
-      return [
-        new Community({
-          id: 3,
-          name: 'Gradido-Akademie',
-          description: 'Freies Institut für Wirtschaftsbionik.',
-          url: 'https://gradido.net',
-          registerUrl: 'https://gdd1.gradido.com/register-community',
-        }),
-      ]
-    return [
-      new Community({
-        id: 1,
-        name: 'Gradido Entwicklung',
-        description: 'Die lokale Entwicklungsumgebung von Gradido.',
-        url: 'http://localhost/',
-        registerUrl: 'http://localhost/register-community',
-      }),
-      new Community({
-        id: 2,
-        name: 'Gradido Staging',
-        description: 'Der Testserver der Gradido-Akademie.',
-        url: 'https://stage1.gradido.net/',
-        registerUrl: 'https://stage1.gradido.net/register-community',
-      }),
-      new Community({
-        id: 3,
-        name: 'Gradido-Akademie',
-        description: 'Freies Institut für Wirtschaftsbionik.',
-        url: 'https://gradido.net',
-        registerUrl: 'https://gdd1.gradido.com/register-community',
-      }),
-    ]
+  async getCommunities(): Promise<Community[]> {
+    const dbCommunities: DbCommunity[] = await DbCommunity.find({
+      order: { foreign: 'ASC', publicKey: 'ASC', apiVersion: 'ASC' },
```

Comment:
Do we want to export both foreign and home community?

Reply:
yes, as long as we have only this community table of federation dynamically filled by the dht-node we show the whole content of this table.
As soon as we will have the second community-Table, which will be different from the community-federation-Table (the migration will rename the current community-table to community-federation), where the permenant community-data will be stored like the community-name/ -uuid/ -etc. there will be a change for the community-List presented in the UI